build-system/common/utils.js

```diff
@@ -111,7 +123,17 @@ function getFilesToCheck(globs, options = {}) {
     }
     return logFiles(filesChanged);
   }
-  return globby.sync(globs, options);
+  const {trackedOnly = false, ...globbyOptions} = options;
```

Comment:
This should be `options` and not `globbyOptions`, right?

Reply:
We can't override `options` by destructuring. In any case, we don't need this anymore.